ant/project.properties

```diff
@@ -9,6 +9,7 @@ project.datadir=qz
 
 install.opts=-Djna.nosys=true
 launch.opts=-Xms512m ${install.opts}
+launch.jigsaw=--add-opens java.desktop/sun.lwawt.macosx=ALL-UNNAMED --add-opens java.desktop/java.awt=ALL-UNNAMED
```

Comment:
Maybe you can use AddOpens MANIFEST.MF entries or module-info.java, not sure 🤓

Reply:
I would much prefere this however I believe forcing `module-info.java` will break compilation with Java < 8 and require a special ant target.  We will probably do this in the near future when we finish bundling the runtime #468.